snap/README.md

```diff
@@ -60,11 +60,11 @@ $ sudo cp /snap/kata-containers/current/usr/share/defaults/kata-containers/confi
 $ $EDITOR /etc/kata-containers/configuration.toml
 ```
 
-## Integration with docker and kubernetes
+## Integration with docker and Kubernetes
 
 The path to the runtime provided by the Kata Containers snap image is
 `/snap/kata-containers/current/usr/bin/kata-runtime`. You should use it to
-run Kata Containers with [docker][9] and [kubernetes][10].
+run Kata Containers with [docker][9] and [Kubernetes][10].
 
 ## Remove snap
```

Comment:
You capitalised the toc, do you need to match the section header (and maybe any of the body) here?
Whilst here, afaict Ubuntu never capitalise 'snap' ?

Reply:
I don't follow? This section heading does match the toc (which I actually lower-cased)...?

Reply:
ay, yep, I read the patch backwards ;-) np